runtime/module.go

```diff
@@ -67,6 +67,7 @@ func init() {
 			ProvideKVStoreKey,
 			ProvideTransientStoreKey,
 			ProvideMemoryStoreKey,
+			ProvideObjectStoreKey,
```

Comment:
do we also need ProvideObjectStoreService?

Reply:
not sure, I don't understand the store service, it adds another layer of wrapper

Reply:
we havent use runtime yet, maybe no need now